pkgs/os-specific/linux/kernel/manual-config.nix

```diff
@@ -239,6 +239,8 @@ lib.makeOverridable (
 
         RUST_LIB_SRC = lib.optionalString withRust rustPlatform.rustLibSrc;
 
+        BINDGEN_EXTRA_CLANG_ARGS = "-Qunused-arguments";
```

Comment:
What's this for? Do we need it for a Nixpkgs-specific reason?

Reply:
https://github.com/NixOS/nixpkgs/pull/390631#issuecomment-2831927650

Reply:
Oh, bindgen is wrapped too. Maybe we should use unwrapped bindgen too?

Reply:
Yes, that works